scripts/run_integration_tests_aarch64.sh

```diff
@@ -59,13 +59,16 @@ build_custom_linux() {
 
     checkout_repo "$LINUX_CUSTOM_DIR" "$LINUX_CUSTOM_URL" "$LINUX_CUSTOM_BRANCH"
 
-    cp $SRCDIR/resources/linux-config-aarch64 $LINUX_CUSTOM_DIR/.config
-
-    pushd $LINUX_CUSTOM_DIR
-    time make -j `nproc`
-    cp arch/arm64/boot/Image "$WORKLOADS_DIR/" || exit 1
-    cp arch/arm64/boot/Image.gz "$WORKLOADS_DIR/" || exit 1
-    popd
+    if [ ! -f "$LINUX_CUSTOM_DIR/.built" ]; then
```

Comment:
This seems like a bad idea. The kernel will not get rebuilt on a config change? Or depending on `checkout_repo` if the repository changes.

Reply:
Aha, yes this is a mistake.

In retrospect, I merged this PR too early to collect enough feedback.